Bottom: 82ccc1598b866dd10eb26d5d64d800db4829fcb0
Top:    4bde380a913783f9c60c684e7e6f060449baffda
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-11-23 16:17:05 +0800

Refresh of add-a-vq-meta-data-iotlb

---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index dbd9561..4c4b16e 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -726,7 +726,7 @@ static void __user *vhost_vq_meta_fetch(struct vhost_virtqueue *vq,
 	if (!node)
 		return NULL;
 
-	BUG_ON(addr < node->start || addr + size >= node->start + node->size);
+	BUG_ON(addr < node->start || addr + size > node->start + node->size);
 
 	return (void *)(node->userspace_addr + (u64)addr - node->start);
 }
